### ISSUE TITLE
fatal glob import error on windows

### ISSUE THREAD

**Issue by mrtristan** [scalyr/scalyr-agent-2#366]
running on windows server 2019, just updated to `2.0.57` from `2.0.53` and getting the below error. bringing it straight here without looking in to it too much as i see there's glob work that's been done in the latest release, so assuming that's related.

```
2020/01/17 15:02:15Z: Message: The errors from user scripts: Unrecognized command-line input arguments: 'token'. For usage refer to: .\config.cmd --help or ./config.sh --help
Invalid configuration provided for token. Terminating unattended configuration.
Traceback (most recent call last):
  File "agent_main.py", line 68, in <module>
  File "scalyr_agent\copying_manager.pyc", line 34, in <module>
  File "scalyr_agent\log_processing.pyc", line 59, in <module>
ImportError: No module named glob2
```

**Comment by ajtcode** [comment]
@mrtristan thank you for bringing this to our attention. I have raised it with our engineering team for further investigation and will provide an update as soon as possible. Please stay tuned.

**Comment by ajtcode** [comment]
@mrtristan I can confirm that this issue has been resolved with the release of v2.0.58. Please install it (https://www.scalyr.com/scalyr-repo/stable/latest/ScalyrAgentInstaller-2.0.58.msi) and let me know if I can be of further assistance.

**Comment by mrtristan** [comment]
@ajtcode appreciate it. i'll bump my packer process to use this new image and will report back once it gets rotated in. will reopen this if it's not solved
